website/content/docs/ecs/resource-usage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+page_title: Resource Usage - AWS ECS
+description: >-
+  Resource usage of the Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Resource Usage
+
+We ran performance tests on Consul on ECS to determine the resource usage of
+the sidecar containers Consul on ECS injects along with the ACL controller.
+[The architecture page](/docs/ecs/architecture) describes each Consul on ECS
+component in depth.
+
+We ran performance tests by deploying clusters of varying sizes and measuring
+resource usage during conditions that stress Consul on ECS components. After
+each iteration of performance testing, we recorded resource usages of each
+component to determine worst-case scenario resource usage in a production
+environment. During performance testing, we used ECS' minimum allowable CPU and memory
+resources, 256 CPU shares and 512 MB of memory, to demonstrate that
+Consul on ECS along with application containers can run on the smallest ECS
+tasks. Here is the maximum resource usage we observed for each container:
```

Comment:
```suggestion
tasks. The following table shows the maximum resource usage we observed for each container:
```

One of my doc learnings from Adam is to make the document read sort of linearly downward. So more explaining what follows, and less "as shown above" where the reader has to jump back up.